Archipelago.HollowKnight/Archipelago.cs

```diff
@@ -275,14 +275,14 @@ public void StartOrResumeGame(bool randomize)
 
                 LogDebug($"StartOrResumeGame: Room: {ApSettings.RoomSeed}; Seed = {ApSettings.RoomSeed}");
 
-                var randomizer = new ArchipelagoRandomizer(loginResult.SlotData);
+                ArchipelagoRandomizer randomizer = new ArchipelagoRandomizer(loginResult.SlotData);
```

Comment:
We can use use shorthand constructors here now `new(loginResult, SlotData)`

Reply:
To expand, I only like this syntax on the same line as the declaration so the ones I've highlighted are the only constructors that need change.

I probably need to invest in a proper style guide and possibly a style checker action